examples/phi2/readme.md

```diff
@@ -1,15 +1,29 @@
 # phi2 optimization with Olive
 This folder contains an example of phi2 optimization with Olive workflow.
 
-- CPU: *PyTorch Model -> Onnx Model -> Transformers Optimized Onnx Model -> Quantized Onnx Model -> ONNX Runtime performance tuning*
+- *PyTorch Model -> Onnx Model -> Transformers Optimized Onnx Model -> Quantized Onnx Model -> ONNX Runtime performance tuning*
 
 ## Prerequisites
 * einops
+* Pytorch>=2.2.0 and ORT nightly. Refer to https://github.com/microsoft/onnxruntime/tree/main/onnxruntime/python/tools/transformers/models/phi2#prerequisites
```

Comment:
Instead of linking the readme from the ort example, could you copy over the instructions for installation instructions for torch and ort-nightly? 
The linked prerequisite section has some more instructions about running the model conversion which might be confusing here. 

Other than that, the rest of the PR looks good to me. Just waiting for the related ORT PR to merge and the nightly package to be published before we merge this one. Thanks!

Reply:
Great! I'll let you know once the ORT PR is in